packages/models-library/src/models_library/api_schemas_dynamic_sidecar/containers.py

```diff
@@ -15,4 +15,7 @@ class ActivityInfo(BaseModel):
     )
 
 
+ActivityInfoAdapter: Final[TypeAdapter[ActivityInfo]] = TypeAdapter(ActivityInfo)
```

Comment:
Is there not something akin to C++ templates for this?

Reply:
Would you mind clarifying this? Could I have an example of what you are searching for?